386/bin/auth/disable

```diff
@@ -1,5 +0,0 @@
-#!/bin/rc
```

Comment:
but is this alsoi in amd64? this is a very handy script. 
is it in /rc/bin? 
this is really messed up, that rc scripts are in /386/bin